pybossa/auth/auditlog.py

```diff
@@ -35,10 +35,14 @@ def _create(self, user, auditlog, project_id=None):
         return False
 
     def _read(self, user, auditlog=None, project_id=None):
-        if user.is_anonymous or (auditlog is None and project_id is None):
+        if user.is_anonymous or (auditlog is None and project_id is None and not user.admin):
```

Comment:
if we move #41 <if user.admin return True> as first check in this func, check of not user.admin can be removed/simplified.

Reply:
If i move #41 we will check for user.admin before checking user.is_anonymous. Is that ok ?